src/styles/search__label.css

```diff
@@ -0,0 +1,32 @@
+.search__label {
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.search__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search__label_big::after {
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
```

Comment:
The fixed pixel values for `top` and `left` could potentially cause issues if the label size changes. Consider using relative units like percentages or `em` for better scalability.